src/lib/list/selection-list.ts

```diff
@@ -97,11 +97,11 @@ export class MatSelectionListChange {
     '(focus)': '_handleFocus()',
     '(blur)': '_handleBlur()',
     '(click)': '_handleClick()',
-    'tabindex': '-1',
     '[class.mat-list-item-disabled]': 'disabled',
     '[class.mat-list-item-focus]': '_hasFocus',
     '[attr.aria-selected]': 'selected.toString()',
     '[attr.aria-disabled]': 'disabled.toString()',
+    '[attr.tabindex]': 'disabled ? null : -1'
```

Comment:
I'm not sure whether it should be completely un-focusable when it's disabled. If that's the case, users might not know that the list is even there. There are some examples in the a11y guidelines where items are still focusable when disabled, but they don't do anything when clicked. E.g. here's an excerpt from the [menu guidelines](https://www.w3.org/TR/wai-aria-practices-1.1/#menu):

> Disabled menu items are focusable but cannot be activated.

Reply:
That's a good point. How about keeping it focusable? but just ignoring keypress events?

Reply:
Just thought a bit more about it. Even if the disabled list is focusable, the user wouldn't be able to walk through the individual options, and therefore it wouldn't really help knowing that there is a list?

Reply:
True, but the question is whether the user shouldn't be able to tab through the list items, even if they're disabled.

Reply:
I see what you mean. I think allowing to tab through the list items makes sense. We would just disable the toggling behavior.

Also if we do this, then we'd need to find a way for the `ListKeyManager` that still allows the user to navigate to disabled items. e.g. a new option for the list key manager. Thoughts?